.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,7 @@
+contact_links:
+    - name: 📃 Documentation Issue
+      url: https://github.com/reactjs/reactjs.org/issues/new
+      about: This issue tracker is not for documentation issues. Please file documentation issues here.
+    - name: 🤔 Questions and Help
+      url: https://stackoverflow.com/questions/tagged/react
+      about: This issue tracker is not for questions. Please ask questions here.
```

Comment:
Let's change this to:

This issue tracker is not for support questions. Please refer to the React community's help and discussion forums.

Reply:
@necolas Thank you for review it, I reflected this!
https://github.com/facebook/react/pull/18039/commits/c1459ca6ec39fac2637b389ed279125723d34706